notebooks/analysis/units.ipynb

```diff
@@ -0,0 +1,575 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a name=\"top\"></a>\n",
+    "<div style=\"width:1000 px\">\n",
+    "\n",
+    "<div style=\"float:right; width:98 px; height:98px;\">\n",
+    "<img src=\"https://raw.githubusercontent.com/Unidata/MetPy/master/src/metpy/plots/_static/unidata_150x150.png\" alt=\"Unidata Logo\" style=\"height: 98px;\">\n",
+    "</div>\n",
+    "\n",
+    "<h1>Units in MetPy</h1>\n",
+    "<h3>Unidata AMS 2021 Student Conference</h3>\n",
+    "\n",
+    "<div style=\"clear:both\"></div>\n",
+    "</div>\n",
+    "\n",
+    "---\n",
+    "\n",
+    "This notebook will help you become familiar with unit support in the `MetPy` library. Learn how to apply units to values, calculate new values from united values, and use units in `MetPy` functions. `MetPy` provides unit support through the `Pint` library. You can access the documentation for `Pint` by clicking on the image to the right.\n",
+    "<a href=https://pint.readthedocs.io/en/stable/><div style=\"float:right; width:250 px\"><img src=\"https://pint.readthedocs.io/en/stable/_images/logo-full.jpg\" alt=\"logo for the Pint library\" style=\"height: 300px;\"></div></a>\n",
+    "\n",
+    "\n",
+    "### Focuses\n",
+    "* Introduce the use of `units` to track values with physical units\n",
+    "* Demonstrate basic operations with and on values with units\n",
+    "* Show examples of using units in `MetPy`\n",
+    "\n",
+    "\n",
+    "### Objectives\n",
+    "1. [Introduction to Units](#1.-Introduction-to-Units)\n",
+    "1. [Quantity Methods and Attributes](#2.-Quantity-Methods-and-Attributes)\n",
+    "1. [MetPy and Units](#3.-MetPy-and-Units)\n",
+    "1. [Gotchas](#4.-Gotchas-)\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import metpy.calc as mpcalc\n",
+    "from metpy.units import units"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Introduction to Units\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`MetPy` provides units support through the `Pint` library. Units in Python are, as described by the [`Pint` documentation](https://pint.readthedocs.io/en/stable/), \"the product of a numerical value and a unit of measurement.\" They are used to programatically keep track of physical quantities so we can worry less about unit conversion errors and focus on our coding objective instead.\n",
```

Comment:
Is this supposed to say "Units in Python..." or "Values in Python..."?

Reply:
Not quite either, I just worded it poorly. Fix incoming shortly.